themes/Hacks2013/style.css

```diff
@@ -58,7 +58,7 @@ body { position: relative; color: #fac817; font: 10px/1.6 Helvetica, "Helvetica
 .page .post,
 .fail { font-size: 1.4em; }
 
-.post h1, .post h2, .post h3, .post h4, .post h5, .post h6 { margin-bottom: .35em; }
+.post h1, .post h2, .post h3, .post h4, .post h5, .post h6 { margin-bottom: 1em; margin-top: 1em;}
```

Comment:
I think it'll look better for heading to have less margin below than they do above. When the space above and below is equal it's like the heading is just floating in space. Maybe make this <code>margin: 1em 0 .5em;</code>? This connects it visually to the text the heading is introducing.